### PR TITLE
fix: pagination and proof params in openapi spec

### PR DESCRIPTION
**Pull request by agraebe**
fixing #222

### CODE REVIEW

File: docs/openapi.yaml [hunk 1]
```diff
@@ -101,6 +101,26 @@ paths:
...
+          description: index of first transaction to fetch
+          required: false
+          schema:
+            type: integer
+        - name: type
+          in: query
+          description: Filter by transaction type
+          required: false
+          schema:
+            type: string
```
Comment (zone117x) [inline docs/openapi.yaml:122]: This is type of `Array<string>`. Looks like the Express query array pattern is supported by openapi, 
e.g. `?type=coinbase&type=token_transfer`

https://swagger.io/docs/specification/serialization:
image

Comment (zone117x) [reply docs/openapi.yaml:122]: So something like
```
type: array
items:
  type: string
  enum: [coinbase, token_transfer, smart_contract, contract_call, poison_microblock]
```

Comment (agraebe) [reply docs/openapi.yaml]: Cool, updated!

**Review by zone117x — APPROVED** [review]
LGTM. Thanks @agraebe!

**Pull request merged**